specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/1.2-preview/iotcentral.json

```diff
@@ -0,0 +1,5638 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure IoT Central",
+    "description": "Azure IoT Central is a service that makes it easy to connect, monitor, and manage your IoT devices at scale.",
+    "version": "1.2-preview"
```

Comment:
API version should be a date in YYYY-MM-DD format, optionally suffixed with '-preview'.